### PR TITLE
Follow redirect

### PR DESCRIPTION
**Pull request by clesaec**

### CODE REVIEW

File: workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java [hunk 1]
```diff
@@ -24,6 +24,12 @@
     public void configure(Connection connection, ConfigurerConfiguration configuration) {
         log.debug("[configure] [{}] {}", connection.getMethod(), connection.getUrl());
         connection //
-                .withHeader("Content-Type", CONTENT_TYPE_APPLICATION_X_WWW_FORM_URLENCODED).withoutFollowRedirects();
+                .withHeader("Content-Type", CONTENT_TYPE_APPLICATION_X_WWW_FORM_URLENCODED);
+
+        try {
+            connection.withoutFollowRedirects();
+        } catch (NoSuchMethodError ex) {
+            log.warn("can't do withoutFollowRedirects");
```
Comment (undx) [inline workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java]: `withoutFollowRedirects operation unsupported.` ?

Comment (clesaec) [reply workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java]: Yes, when tck version for compilation is not the same as tck version in remote engine.

Comment (undx) [reply workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java]: I was talking about the log.warn message content ;-)

Comment (clesaec) [reply workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java]: different than 'can't do withoutFollowRedirects' ?

Comment (undx) [reply workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java]: a little bit nicer :-)

Comment (clesaec) [reply workday/src/main/java/org/talend/components/workday/service/AccessTokenConfigurer.java]: Changed even if it's same meaning.

**Pull request merged**